brainkeeper/src/app/app.module.ts

```diff
@@ -3,10 +3,14 @@ import { NgModule } from '@angular/core';
 
 import { AppRoutingModule } from './app-routing.module';
 import { AppComponent } from './app.component';
+import { GridComponent } from './components/session_screen/grid/grid.component';
+import { PersonNameComponent } from './components/session_screen/person-name/person-name.component';
 
 @NgModule({
   declarations: [
-    AppComponent
+    AppComponent,
+    GridComponent,
+    PersonNameComponent
```

Comment:
please always use a trailing commas when inserting new components in order to decrease number of changed lines and reduce number of merge conflicts